docs/src/scs/sbom.md

```diff
@@ -0,0 +1,34 @@
+# Software bill of materials (SBOM)
+Software bill of materials (SBOM) is a formal, machine readable document that provides a list of software components that make up the target software and all its dependencies.
```

Comment:
**A software** ... **machine-readable** ...

Reply:
Resolved in https://github.com/tiiuae/ghaf/pull/46/commits/50a79b10dcbefd55fef9dfd3a7dacf8e4a1fafc7